seed/models/property_measures.py

```diff
@@ -57,6 +60,9 @@ class PropertyMeasure(models.Model):
         (SCALE_MULTIPLE_PREMISES, 'Multiple premises'),
         (SCALE_ENTIRE_FACILITY, 'Entire facility'),
         (SCALE_ENTIRE_SITE, 'Entire site'),
+        (SCALE_ENTIRE_BUILDING, 'Entire building'),
+        (SCALE_COMMON_AREAS, 'Common areas'),
+        (SCALE_TENANT_AREAS, 'Tenant areas'),
```

Comment:
I think you need to run `./manage.py makemigrations` to create the updates needed for these changes.